languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/replace.txt

```diff
@@ -578,4 +578,17 @@ Alex Grimaldos=Álex Grimaldos
 Joko Winterscheid=Joko Winterscheidt
 Joachim Winterscheid=Joachim Winterscheidt
 Joko Winterscheids=Joko Winterscheidts
-Joachim Winterscheids=Joachim Winterscheidts
\ No newline at end of file
+Joachim Winterscheids=Joachim Winterscheidts
+sähe=säe	Das Verb 'säen' (etw. anpflanzen) wird ohne 'h' geschrieben; es sei denn Sie meinen den Konjunktiv II von 'sehen'.
```

Comment:
ja, kann man mal ausprobieren. Weiß nicht, wie oft die Leute tatsächlich den Konjunktiv von sehen schreiben wollen. Man kann nicht sehen, wie einzelne Wörter aus der replace angenommen oder disabled werde, oder?

Reply:
Doch mit rule_id=`DE_SIMPLE_REPLACE_xxx` example: https://internal1.languagetool.org/grafana/d/BY_CNDHGz/rule-events-analysis?orgId=1&var-rule_id=DE_SIMPLE_REPLACE_SAGRADA_FAMILIA&var-language=All&from=now-90d&to=now